src/www/pages/patterns/header-and-footer/index.njk

```diff
@@ -112,6 +112,21 @@
   ])
 }}
 
+{# Full header #}
+<h3>Full variant of header</h3>
+
+{{
+  compExample([
+    wmndsHeader({
+      navItems: navItems,
+      banner: true,
+      breadcrumbs: {
+        section: section,
+        pageTitle: pageTitle
+      }
+    })
+  ])
+}}
```

Comment:
Can you move this example to the top of the page as this is our preferred header that we want people to use.